contracts/Otoken.sol

```diff
@@ -0,0 +1,68 @@
+/*
+ * Otoken contract. Copyright © 2020 by Opyn.co .
+ * Author: Opyn
+ */
+pragma solidity 0.6.10;
+
+/**
+ * @title Otoken
+ * @dev TODO
+ */
+
+contract Otoken {
+    /** TODO: */
+    bool public isPut;
+
+    /** TODO: */
+    uint256 public strikePrice;
+
+    /** TODO: */
+    uint256 public expiryTimestamp;
+
+    /** TODO: */
+    address public underlying;
+
+    /** TODO: */
+    address public strike;
+
+    /** TODO: */
+    address public collateral;
+
+    /** TODO: */
+    address public addressBook;
+
+    constructor() internal {
+        /** TODO: */
+    }
+
+    modifier onlyFactory() {
+        /** TODO: */
+        _;
+    }
+
+    modifier onlyController() {
+        /** TODO: */
+        _;
+    }
+
+    /**
+     * @notice TODO:
+     * @dev TODO:
+     * @param underlyingAsset TODO:
+     * @param strikeAsset TODO:
+     * @param collateralAsset TODO:
+     * @param isPut TODO:
+     * @param expiryTimestamp TODO:
+     * @param strikePrice TODO:
+     */
+    function init(
+        address underlyingAsset,
+        address strikeAsset,
+        address collateralAsset,
+        bool isPut,
+        uint256 expiryTimestamp,
+        uint256 strikePrice
```

Comment:
How do people feel about the order of the parameters? I think we should probably somehow follow our naming
`ETHUSDC/{timestamp}/205P/USDC`, so I think it's better to use: `underlyingAsset, strikeAsset, collateralAsset, expiryTimestamp, strikePrice, isPut`